File: MWR/Source/MWRSolution/MWRDBModule/DBModule/TblMWInvAuthorize.cs
using System;
using System.Collections.Generic;
using System.Text;
using ComLib.db;

namespace YRKJ.MWR
{

    public class TblMWInvAuthorize : BaseDataModule
    {

        private static string TableName = "MWInvAuthorize";
        public TblMWInvAuthorize()
        {
        }
        public static string getFormatTableName()
        {
            return SqlCommonFn.FormatSqlTableNameString(TableName);
        }

        public static DataColumnInfo[] Columns = 
                new DataColumnInfo[]{
            new DataColumnInfo(true,false,false,false,"InvAuthId",SqlCommonFn.DataColumnType.INT,10),
            new DataColumnInfo(false,true,false,false,"TxnNum",SqlCommonFn.DataColumnType.STRING,20),
            new DataColumnInfo(false,true,false,false,"TxnDetailId",SqlCommonFn.DataColumnType.INT,10),
            new DataColumnInfo(false,true,false,false,"EmpyCode",SqlCommonFn.DataColumnType.STRING,20),
            new DataColumnInfo(false,true,false,false,"EmpyName",SqlCommonFn.DataColumnType.STRING,45),
            new DataColumnInfo(false,true,false,false,"WSCode",SqlCommonFn.DataColumnType.STRING,20),
            new DataColumnInfo(false,true,false,false,"AuthEmpyCode",SqlCommonFn.DataColumnType.STRING,20),
            new DataColumnInfo(false,true,false,false,"AuthEmpyName",SqlCommonFn.DataColumnType.STRING,45),
            new DataColumnInfo(false,true,false,false,"Remark",SqlCommonFn.DataColumnType.STRING,45),
            new DataColumnInfo(false,true,false,false,"SubWeight",SqlCommonFn.DataColumnType.DECIMAL,10),
            new DataColumnInfo(false,true,false,false,"TxnWeight",SqlCommonFn.DataColumnType.DECIMAL,10),
            new DataColumnInfo(false,true,false,false,"DiffWeight",SqlCommonFn.DataColumnType.DECIMAL,10),
            new DataColumnInfo(false,true,false,false,"EntryDate",SqlCommonFn.DataColumnType.DATETIME,10),
            new DataColumnInfo(false,true,false,false,"CompDate",SqlCommonFn.DataColumnType.DATETIME,0),
            new DataColumnInfo(false,true,false,false,"Status",SqlCommonFn.DataColumnType.STRING,2)
        };

        public static DataColumnInfo getInvAuthIdColumn()
        {
            return Columns[0];
        }
        public static DataColumnInfo getTxnNumColumn()
        {
            return Columns[1];
        }
        public static DataColumnInfo getTxnDetailIdColumn()
        {
            return Columns[2];
        }
        public static DataColumnInfo getEmpyCodeColumn()
        {
            return Columns[3];
        }
        public static DataColumnInfo getEmpyNameColumn()
        {
            return Columns[4];
        }
        public static DataColumnInfo getWSCodeColumn()
        {
            return Columns[5];
        }
        public static DataColumnInfo getAuthEmpyCodeColumn()
        {
            return Columns[6];
        }
        public static DataColumnInfo getAuthEmpyNameColumn()
        {
            return Columns[7];
        }
        public static DataColumnInfo getRemarkColumn()
        {
            return Columns[8];
        }
        public static DataColumnInfo getSubWeightColumn()
        {
            return Columns[9];
        }
        public static DataColumnInfo getTxnWeightColumn()
        {
            return Columns[10];
        }
        public static DataColumnInfo getDiffWeightColumn()
        {
            return Columns[11];
        }
        public static DataColumnInfo getEntryDateColumn()
        {
            return Columns[12];
        }
        public static DataColumnInfo getCompDateColumn()
        {
            return Columns[13];
        }
        public static DataColumnInfo getStatusColumn()
        {
            return Columns[14];
        }

        private int _InvAuthId = 0;
        private string _TxnNum = "";
        private int _TxnDetailId = 0;
        private string _EmpyCode = "";
        private string _EmpyName = "";
        private string _WSCode = "";
        private string _AuthEmpyCode = "";
        private string _AuthEmpyName = "";
        private string _Remark = "";
        private decimal _SubWeight = 0;
        private decimal _TxnWeight = 0;
        private decimal _DiffWeight = 0;
        private DateTime _EntryDate = DateTime.MinValue;
        private DateTime _CompDate = DateTime.MinValue;
        private string _Status = "";

        public int InvAuthId
        {
            get
            {
                return _InvAuthId;
            }
            set
            {
                _InvAuthId = value;
            }
        }
        public string TxnNum
        {
            get
            {
                return _TxnNum;
            }
            set
            {
                _TxnNum = value;
            }
        }
        public int TxnDetailId
        {
            get
            {
                return _TxnDetailId;
            }
            set
            {
                _TxnDetailId = value;
            }
        }
        public string EmpyCode
        {
            get
            {
                return _EmpyCode;
            }
            set
            {
                _EmpyCode = value;
            }
        }
        public string EmpyName
        {
            get
            {
                return _EmpyName;
            }
            set
            {
                _EmpyName = value;
            }
        }
        public string WSCode
        {
            get
            {
                return _WSCode;
            }
            set
            {
                _WSCode = value;
            }
        }
        public string AuthEmpyCode
        {
            get
            {
                return _AuthEmpyCode;
            }
            set
            {
                _AuthEmpyCode = value;
            }
        }
        public string AuthEmpyName
        {
            get
            {
                return _AuthEmpyName;
            }
            set
            {
                _AuthEmpyName = value;
            }
        }
        public string Remark
        {
            get
            {
                return _Remark;
            }
            set
            {
                _Remark = value;
            }
        }
        public decimal SubWeight
        {
            get
            {
                return _SubWeight;
            }
            set
            {
                _SubWeight = value;
            }
        }
        public decimal TxnWeight
        {
            get
            {
                return _TxnWeight;
            }
            set
            {
                _TxnWeight = value;
            }
        }
        public decimal DiffWeight
        {
            get
            {
                return _DiffWeight;
            }
            set
            {
                _DiffWeight = value;
            }
        }
        public DateTime EntryDate
        {
            get
            {
                return _EntryDate;
            }
            set
            {
                _EntryDate = value;
            }
        }
        public DateTime CompDate
        {
            get
            {
                return _CompDate;
            }
            set
            {
                _CompDate = value;
            }
        }
        public string Status
        {
            get
            {
                return _Status;
            }
            set
            {
                _Status = value;
            }
        }

         public override void SetValue(System.Data.DataRow row)
         {
             SetValue(ref _InvAuthId, row["InvAuthId"]);
             SetValue(ref _TxnNum, row["TxnNum"]);
             SetValue(ref _TxnDetailId, row["TxnDetailId"]);
             SetValue(ref _EmpyCode, row["EmpyCode"]);
             SetValue(ref _EmpyName, row["EmpyName"]);
             SetValue(ref _WSCode, row["WSCode"]);
             SetValue(ref _AuthEmpyCode, row["AuthEmpyCode"]);
             SetValue(ref _AuthEmpyName, row["AuthEmpyName"]);
             SetValue(ref _Remark, row["Remark"]);
             SetValue(ref _SubWeight, row["SubWeight"]);
             SetValue(ref _TxnWeight, row["TxnWeight"]);
             SetValue(ref _DiffWeight, row["DiffWeight"]);
             SetValue(ref _EntryDate, row["EntryDate"]);
             SetValue(ref _CompDate, row["CompDate"]);
             SetValue(ref _Status, row["Status"]);
         }


    }
}

File: DemoApp/TblModel/TblMWInvAuthorize.java
package com.yrkj.ar.bean;
import com.yrkj.lib.db.BaseDataModule;
import com.yrkj.lib.db.DataColumnInfo;
import com.yrkj.lib.db.SqlCommonFn;
import com.yrkj.lib.db.SqlCommonFn.DataColumnType;

public class TblMWInvAuthorize extends BaseDataModule {
    private static String TableName = "MWInvAuthorize";
    public TblMWInvAuthorize(){
    }
    public static String getFormatTableName(){
        return SqlCommonFn.FormatSqlTableNameString(TableName);
    }

    public static DataColumnInfo[] Columns = 
            new DataColumnInfo[]{
        new DataColumnInfo(true,false,false,false,"InvAuthId",DataColumnType.INT,10),
        new DataColumnInfo(false,true,false,false,"TxnNum",DataColumnType.STRING,20),
        new DataColumnInfo(false,true,false,false,"TxnDetailId",DataColumnType.INT,10),
        new DataColumnInfo(false,true,false,false,"EmpyCode",DataColumnType.STRING,20),
        new DataColumnInfo(false,true,false,false,"EmpyName",DataColumnType.STRING,45),
        new DataColumnInfo(false,true,false,false,"WSCode",DataColumnType.STRING,20),
        new DataColumnInfo(false,true,false,false,"AuthEmpyCode",DataColumnType.STRING,20),
        new DataColumnInfo(false,true,false,false,"AuthEmpyName",DataColumnType.STRING,45),
        new DataColumnInfo(false,true,false,false,"Remark",DataColumnType.STRING,45),
        new DataColumnInfo(false,true,false,false,"SubWeight",DataColumnType.DECIMAL,10),
        new DataColumnInfo(false,true,false,false,"TxnWeight",DataColumnType.DECIMAL,10),
        new DataColumnInfo(false,true,false,false,"DiffWeight",DataColumnType.DECIMAL,10),
        new DataColumnInfo(false,true,false,false,"EntryDate",DataColumnType.STRING,10),
        new DataColumnInfo(false,true,false,false,"CompDate",DataColumnType.STRING,0),
        new DataColumnInfo(false,true,false,false,"Status",DataColumnType.STRING,2)
    };

    public static DataColumnInfo getInvAuthIdColumn(){
        return Columns[0];
    }
    public static DataColumnInfo getTxnNumColumn(){
        return Columns[1];
    }
    public static DataColumnInfo getTxnDetailIdColumn(){
        return Columns[2];
    }
    public static DataColumnInfo getEmpyCodeColumn(){
        return Columns[3];
    }
    public static DataColumnInfo getEmpyNameColumn(){
        return Columns[4];
    }
    public static DataColumnInfo getWSCodeColumn(){
        return Columns[5];
    }
    public static DataColumnInfo getAuthEmpyCodeColumn(){
        return Columns[6];
    }
    public static DataColumnInfo getAuthEmpyNameColumn(){
        return Columns[7];
    }
    public static DataColumnInfo getRemarkColumn(){
        return Columns[8];
    }
    public static DataColumnInfo getSubWeightColumn(){
        return Columns[9];
    }
    public static DataColumnInfo getTxnWeightColumn(){
        return Columns[10];
    }
    public static DataColumnInfo getDiffWeightColumn(){
        return Columns[11];
    }
    public static DataColumnInfo getEntryDateColumn(){
        return Columns[12];
    }
    public static DataColumnInfo getCompDateColumn(){
        return Columns[13];
    }
    public static DataColumnInfo getStatusColumn(){
        return Columns[14];
    }

    public int InvAuthId = 0;
    public String TxnNum = "";
    public int TxnDetailId = 0;
    public String EmpyCode = "";
    public String EmpyName = "";
    public String WSCode = "";
    public String AuthEmpyCode = "";
    public String AuthEmpyName = "";
    public String Remark = "";
    public decimal SubWeight = 0;
    public decimal TxnWeight = 0;
    public decimal DiffWeight = 0;
    public String EntryDate = "";
    public String CompDate = "";
    public String Status = "";

    public int getInvAuthId() {
        return  InvAuthId;
    }
    public void setInvAuthId(int InvAuthId) {
        this.InvAuthId = InvAuthId;
    }
    public String getTxnNum() {
        return  TxnNum;
    }
    public void setTxnNum(String TxnNum) {
        this.TxnNum = TxnNum;
    }
    public int getTxnDetailId() {
        return  TxnDetailId;
    }
    public void setTxnDetailId(int TxnDetailId) {
        this.TxnDetailId = TxnDetailId;
    }
    public String getEmpyCode() {
        return  EmpyCode;
    }
    public void setEmpyCode(String EmpyCode) {
        this.EmpyCode = EmpyCode;
    }
    public String getEmpyName() {
        return  EmpyName;
    }
    public void setEmpyName(String EmpyName) {
        this.EmpyName = EmpyName;
    }
    public String getWSCode() {
        return  WSCode;
    }
    public void setWSCode(String WSCode) {
        this.WSCode = WSCode;
    }
    public String getAuthEmpyCode() {
        return  AuthEmpyCode;
    }
    public void setAuthEmpyCode(String AuthEmpyCode) {
        this.AuthEmpyCode = AuthEmpyCode;
    }
    public String getAuthEmpyName() {
        return  AuthEmpyName;
    }
    public void setAuthEmpyName(String AuthEmpyName) {
        this.AuthEmpyName = AuthEmpyName;
    }
    public String getRemark() {
        return  Remark;
    }
    public void setRemark(String Remark) {
        this.Remark = Remark;
    }
    public decimal getSubWeight() {
        return  SubWeight;
    }
    public void setSubWeight(decimal SubWeight) {
        this.SubWeight = SubWeight;
    }
    public decimal getTxnWeight() {
        return  TxnWeight;
    }
    public void setTxnWeight(decimal TxnWeight) {
        this.TxnWeight = TxnWeight;
    }
    public decimal getDiffWeight() {
        return  DiffWeight;
    }
    public void setDiffWeight(decimal DiffWeight) {
        this.DiffWeight = DiffWeight;
    }
    public String getEntryDate() {
        return  EntryDate;
    }
    public void setEntryDate(String EntryDate) {
        this.EntryDate = EntryDate;
    }
    public String getCompDate() {
        return  CompDate;
    }
    public void setCompDate(String CompDate) {
        this.CompDate = CompDate;
    }
    public String getStatus() {
        return  Status;
    }
    public void setStatus(String Status) {
        this.Status = Status;
    }



}
